index.html

```diff
@@ -288,11 +289,9 @@ <h3>
           </li>
         </ol>
         <p data-link-for="VisibilityState">
-          To accommodate assistive technologies that are typically full screen
-          but still show a view of the page, when applicable, on getting, the
-          <a data-link-for="Document">visibilityState</a> attribute MAY return
-          "<a>visible</a>", instead of "<a>hidden</a>", when the user agent is
-          not minimized but is fully obscured by other applications.
+          When possible, if the user agent is fully obscured by another 
```

Comment:
Maybe replace "When possible" with something more concrete?
There are 2 conditions here:
* The UA is aware that it is being obscured.
* The UA is aware that it is being viewed in an accessibility tool.

Only when it is aware of both, it SHOULD return specific values.
It may also make sense to define what the behavior MAY be when the UA is aware of one but not the other.

Reply:
Case 2 of "visible" in the paragraphs above has already covered a MUST statement when both of these two conditions are met.

Does it make sense to say:

When possible, if the UA is aware that it is being obscured, it should return "visible", as long as it is aware that it is being viewed in an accessibility tool and a view of the doc is displayed in an accessibility tool; (does it matter whether it's fully obscured or not?)

When the UA is aware that it is being viewed in an accessibility tool and a view of the doc is displayed in an accessibility tool, it should return "visible" as long as the doc is the foreground tab.

Reply:
Hmm. Maybe we can drop the paragraph below the processing model and change the processing model instead?

We have the following cases:
* When the UA is not being obscured - "visible"
    - That seems to be covered by the processing model and doesn't require any extra notes/clauses
* When the UA is being obscured, but not viewed by an accessibility tool
    - I'd think this should be "hidden", unless the document can be viewed by accessibility tools without the UA being aware of it.
* When the UA is being obscured *and* viewed by an accessibility tool
    - That's almost covered by the current processing model. I think we should change the wordings of step 2.2 somewhat to something like "The user agent is fully obscured, but a view of the doc is displayed in an accessibility tool."

@toddreifsteck - thoughts?

Reply:
BTW, I like "when possible" because it is literally not always possible to detect full occlusion on all platforms, we are now able to detect this on a couple of platforms and would like to return hidden for those.